config/servers.yml

```diff
@@ -387,3 +387,14 @@ endpoints:
 #         backOffMaxInterval:        10000
 #         backOffMultiplier:         2.0
 #         maxAttempts:               5
+
+---
+# Kryo uses references by default. This feature must be enabled unless we know the payload types being serialized do not
+# contain circular refefences. Disabling this feature improves performance if it is not needed.
```

Comment:
Though commented, the default value is set to `false` (which makes it performant). This might confuse the user about what is mentioned in the description: `Disabling this feature improves performance if it is not needed.` -> whether to set to true or false to make it performant.